src/pages/workspace/WorkspaceResetBankAccountModal.js

```diff
@@ -0,0 +1,75 @@
+import _ from 'underscore';
+import lodashGet from 'lodash/get';
+import PropTypes from 'prop-types';
+import React from 'react';
+import {withOnyx} from 'react-native-onyx';
+import ConfirmModal from '../../components/ConfirmModal';
+import {cancelResetFreePlanBankAccount, resetFreePlanBankAccount} from '../../libs/actions/BankAccounts';
+import withLocalize, {withLocalizePropTypes} from '../../components/withLocalize';
+import reimbursementAccountPropTypes from '../ReimbursementAccount/reimbursementAccountPropTypes';
+import compose from '../../libs/compose';
+import ONYXKEYS from '../../ONYXKEYS';
+import bankAccountPropTypes from '../../components/bankAccountPropTypes';
+import Text from '../../components/Text';
+import styles from '../../styles/styles';
+import BankAccount from '../../libs/models/BankAccount';
+
+const propTypes = {
+    /** Reimbursement account data */
+    reimbursementAccount: reimbursementAccountPropTypes,
+
+    /** List of bank accounts */
+    bankAccountList: PropTypes.arrayOf(bankAccountPropTypes),
+
+    ...withLocalizePropTypes,
+};
+
+const defaultProps = {
+    reimbursementAccount: {},
+    bankAccountList: [],
+};
+
+const WorkspaceResetBankAccountModal = (props) => {
+    const isInOpenState = lodashGet(props.reimbursementAccount, 'achData.state') === BankAccount.STATE.OPEN;
```

Comment:
NAB 
```suggestion
    const isAccountOpen = lodashGet(props.reimbursementAccount, 'achData.state') === BankAccount.STATE.OPEN;
```

Reply:
I'm gonna leave this because I think it's fine for this variable to reference the "state" explicitly here than refer to an "open account" which is slightly more ambiguous (even if it kinda sounds more like something a human would say)